scripts/commands/getta.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
```

Comment:
This GM command will not function.  Needs to be updated to used the caching format

Reply:
it definitely functions on my local, which command can i use as an example of the caching format?

Reply:
All of them

Reply:
See #4418

Reply:
Wow. Apologies. No idea why my local has the old format. I'm sure it's due to not doing a clean rebase pull when switching from ASB and LSB